### DISCUSSION TITLE
Klipperscreen variant: Do we have Filament Runout Sensor feature working?

### DISCUSSION THREAD

**Discussion by DougJoseph** [xblax/flashforge_ad5m_klipper_mod#97] [Q&A]
I have the Klipperscreen variant. I enabled the Filament Runout Sensor feature in the Klipperscreen GUI. However, it has a red bar showing under its name. My filament ran out, but the printer kept going. Just curious if this is supposed to work, or is working for others. Thanks for the awesome mod!

**Comment by xblax** [comment]
Probably your printer config didn't have the fix we already made:
https://github.com/xblax/flashforge_adm5_klipper_mod/issues/50

Will be in the next release (maybe tonight).

**Reply by DougJoseph** [reply]
Awesome, thank you! I will click into that item and read up on it.
